src/Item_Project.php

```diff
@@ -265,16 +267,97 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
 
         switch ($item->getType()) {
             case 'Project':
                 self::showForProject($item);
                 break;
 
             default:
-               // Not defined and used now
-               // Project::showListForItem($item);
+                if (in_array($item->getType(), $CFG_GLPI["project_asset_types"])) {
+                    self::showForAsset($item);
+                }
+                break;
         }
         return true;
     }
+
+    public static function showForAsset(CommonDBTM $item)
+    {
+
+        $item_project = new self();
+        $item_projects = $item_project->find([
+            'itemtype' => $item->getType(),
+            'items_id' => $item->getID()
+        ]);
+
+        $used = $entries = [];
+
+        foreach ($item_projects as $value) {
+            $used[] = $value['projects_id'];
+            $project = new Project();
+            $project->getFromDB($value['projects_id']);
+
+            $priority = CommonITILObject::getPriorityName($project->fields['priority']);
+            $state = ProjectState::getById($project->fields['projectstates_id']);
+
+            if (!empty($project->fields)) {
```

Comment:
`$project->fields` are not supposed to be empty. In such a case, code would fail on the two previous lines..